kura/meta_cluster.py

```diff
@@ -86,9 +86,9 @@ def __init__(
         **kwargs,  # For future use
     ):
         if clustering_model is None:
-            from kura.k_means import KmeansClusteringMethod
+            from kura.cluster import KmeansClusteringModel
```

Comment:
log debug that we're defaulting

Reply:
Will be refactoring this in the second PR